conda-store-server/conda_store_server/server/auth.py

```diff
@@ -35,6 +35,7 @@
 from conda_store_server import api
 from conda_store_server._internal import environment, orm, schema, utils
 from conda_store_server._internal.server import dependencies
+from conda_store_server.server import schema as auth_schema
```

Comment:
I think this module being called `conda_store_server.server` is pretty misleading. This is left over from some other related work around moving modules into an `_internal` module. Punting on renaming this for now.

Reply:
I agree we should leave this alone for now - but it's not a helpful name.